main.css

```diff
@@ -31,99 +34,87 @@
   padding: 8px;
   border: none;
 }
+
```

Comment:
Last but not least, kindly fix the white space on the right side of your mobile view:

![image](https://user-images.githubusercontent.com/38088818/164551128-6651abe6-eade-48a7-9285-ab5ffe564afa.png)